### PR TITLE
docs: Update reference docs and add missing params

### PR DESCRIPTION
**Pull request by agnieszka-m**
### Related Issues

[#3118](https://github.com/deepset-ai/haystack-hub-api/issues/3118)

### Checklist

- [ ] I have updated the referenced issue with new insights and changes
- [ ] If this is a code change, I have added unit tests
- [ ] I've used the [conventional commit specification](https://www.conventionalcommits.org/en/v1.0.0/) for my PR title
- [ ] I updated the docstrings
- [ ] If this is a code change, I added meaningful logs and prepared Datadog visualizations and alerts

### CODE REVIEW

File: deepset_cloud_sdk/cli.py [hunk 1]
```diff
@@ -66,17 +78,19 @@ def list_files(
...
 ) -> None:
-    """List files in deepset Cloud.
-
-    A CLI method to list files that exist in deepset Cloud.
+    """List files that exist in the specified deepset Cloud workspace.
 
     :param api_key: deepset Cloud API key to use for authentication.
     :param api_url: API URL to use for authentication.
-    :param workspace_name: Name of the workspace to list the files from.
+    :param workspace_name: Name of the workspace to list the files from. Uses the workspace from the .EVN file by default.
```
Comment (ArzelaAscoIi) [inline deepset_cloud_sdk/cli.py:85]: Our linting is not happy about this: https://github.com/deepset-ai/deepset-cloud-sdk/actions/runs/5344918400/jobs/9689892588?pr=71#step:5:19

The descripts of params are required to be less than 120 chars by default python standards. Technically we can increase it. I will double check whats the best practice here.

File: deepset_cloud_sdk/__about__.py [hunk 2]
```diff
@@ -1,3 +1,3 @@
 """This file defines the package version."""
 
-__version__ = "0.0.24"
+__version__ = "0.0.25"
```
Comment (ArzelaAscoIi) [inline deepset_cloud_sdk/__about__.py:3]: Bump version to updated docs/docstrings

File: deepset_cloud_sdk/cli.py (hunk 1 above)
Comment (agnieszka-m) [reply deepset_cloud_sdk/cli.py:85]: I'd vote for let's make it as long as needed for it to be clear :)

Comment (ArzelaAscoIi) [reply deepset_cloud_sdk/cli.py:85]: Increased it to 150 chars

**Review by ArzelaAscoIi — APPROVED** [review]
👍 Thanks!

**Pull request merged**